server/core/src/server_utilities.cpp

```diff
@@ -50,7 +50,7 @@ namespace irods
         const auto buf_size = _s.length() + 1;
 
         auto* buf = static_cast<char*>(allocate(sizeof(char) * buf_size));
-        std::strncpy(buf, _s.data(), _s.length());
+        std::snprintf(buf, _s.length(), "%s", _s.data());
```

Comment:
This will truncate the string being copied by one byte. Replacing `_s.length()` with `buf_size` should take care of it.

Reply:
Ignore this one. I was doing the review one commit at a time.

Reply:
Yep, fixed.